### PR TITLE
CMA edits per Zbynek Roubalik

### PR DESCRIPTION
**Pull request by mburke5678**
Zbynek reviewed the CMA docs and [commented in Slack](https://redhat-internal.slack.com/archives/D02TJ37CYSE/p1682526488418329).

Previews:
[Automatically scaling pods based on custom metrics](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html) Added TP note for scaled jobs; removed TP note for CMA

[Understanding how to add custom metrics autoscalers](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-adding_nodes-pods-autoscaling-custom). Edited second sentence

[Understanding the custom metrics autoscaler](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-about_nodes-pods-autoscaling-custom). Edited Note

[Adding a custom metrics autoscaler to a job](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-creating-job_nodes-pods-autoscaling-custom). Added TP note for scaled jobs

[Using trigger authentications](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-creating-trigger-auth_nodes-pods-autoscaling-custom). Cleaned up code example in step 2.

[Adding a custom metrics autoscaler to a workload](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-creating-workload_nodes-pods-autoscaling-custom). Cleaned up code example in step 1.

[Pausing the custom metrics autoscaler for a workload](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-pausing_nodes-pods-autoscaling-custom). Cleaned up code example in step 2. 

[Understanding the custom metrics autoscaler triggers](https://59331--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-trigger_nodes-pods-autoscaling-custom). Cleaned up code for the Prometheus, CPU, memory, and Kafka examples. Added note about the need to use quoted string values to call-outs.  

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@zroubalik PTAL

File: modules/nodes-pods-autoscaling-custom-creating-trigger-auth.adoc [hunk 1]
```diff
@@ -70,9 +70,9 @@ metadata:
 spec:
   scaleTargetRef:
     name: example-deployment
-  maxReplicaCount: 100
-  minReplicaCount: 0
-  pollingInterval: 30
+  maxReplicaCount: '100'
+  minReplicaCount: '0'
+  pollingInterval: '30'
   triggers:
```
Comment (zroubalik) [inline modules/nodes-pods-autoscaling-custom-creating-trigger-auth.adoc:76]: please revert this, the quotoes are only needed for values in `spec.triggers.metadata`

File: modules/nodes-pods-autoscaling-custom-creating-workload.adoc [hunk 2]
```diff
@@ -97,13 +97,13 @@ spec:
...
+        maxAge: '2'
+        maxBackup: '1'
+        maxSize: '50'
   fallback: <11>
-    failureThreshold: 3
-    replicas: 6
-  pollingInterval: 30 <12>
+    failureThreshold: '3'
+    replicas: '6'
+  pollingInterval: '30' <12>
```
Comment (zroubalik) [inline modules/nodes-pods-autoscaling-custom-creating-workload.adoc:106]: please revert this, the quotoes are only needed for values in `spec.triggers.metadata`

File: modules/nodes-pods-autoscaling-custom-creating-workload.adoc [hunk 3]
```diff
@@ -113,8 +113,8 @@ spec:
           stabilizationWindowSeconds: 300
           policies:
           - type: Percent
-            value: 100
-            periodSeconds: 15
+            value: '100'
+            periodSeconds: '15'
```
Comment (zroubalik) [inline modules/nodes-pods-autoscaling-custom-creating-workload.adoc:117]: please revert this, the quotoes are only needed for values in `spec.triggers.metadata`

File: modules/nodes-pods-autoscaling-custom-pausing.adoc [hunk 4]
```diff
@@ -39,13 +39,13 @@ apiVersion: keda.sh/v1alpha1
 kind: ScaledObject
 metadata:
   annotations:
-    autoscaling.keda.sh/paused-replicas: "4" <1>
-  creationTimestamp: "2023-02-08T14:41:01Z"
-  generation: 1
+    autoscaling.keda.sh/paused-replicas: '4' <1>
+  creationTimestamp: '2023-02-08T14:41:01Z'
+  generation: '1'
```
Comment (zroubalik) [inline modules/nodes-pods-autoscaling-custom-pausing.adoc:44]: please revert this, the quotoes are only needed for values in `spec.triggers.metadata`

**Comment by mburke5678** [comment]
@weinliu PTAL

**Comment by weinliu** [comment]
Hi Mike, the updates /lgtm

**Comment by stevsmit** [comment]
@mburke5678 Great work. 2 things I pointed out that are not deal breaking to merge.

**Pull request merged**